paddle/framework/backward.cc

```diff
@@ -375,10 +376,18 @@ std::vector<std::unique_ptr<OpDescBind>> MakeBlockBackward(
     backward_descs.insert(backward_descs.begin() + p.first + 1,
                           std::move(p.second));
   }
+
+  // insert fill one op for target
+  std::unique_ptr<OpDescBind> fill_one_op(new OpDescBind(
```

Comment:
It should be in `AppendBackward`. That logic is wrong.

Reply:
done